content/03.methods.md

```diff
@@ -88,6 +100,7 @@ We used Manta SV [@doi:10/gf3ggb] v1.4.0 for structural variant (SV) calls.
 Manta SV calling was also limited to regions used in Strelka2.
 We also ran LUMPY SV [@doi:10/gf3ggc] v0.2.13 in express mode using default parameters. 
 The hg38 reference used was also limited to canonical chromosome regions.
+The [somatic DNA workflow](https://github.com/kids-first/kf-somatic-workflow) for SNV, INDEL, copy number, and SV calling can be found in the D3b Github repository.
```

Comment:
Probably best to make the link happen at the D3b GitHub repository part of the sentence, since that's where it'd probably be most natural - or link it at both places.

Reply:
Agree with @cgreene.